pkgs/jovian-stubs/sudo

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+echo "[JOVIAN] sudo: stub!"
+
+declare -a final
+
+positional=""
+for value in "$@"; do
+    if [[ -n "$positional" ]]; then
+        final+=("$value")
+    elif [[ "$value" == "-n" ]]; then
+        :
+    else
+        positional="y"
+        final+=("$value")
+    fi
```

Comment:
I suppose I'm missing something from the initial WIP, but did this end-up needlessly doing the same thing in the first and last branch?

Reply:
I didn't change this bit at all.

Reply:
But yes.